index.html

```diff
@@ -231,10 +231,52 @@ <h4>About me</h4>
         </p>        
         <form action="">
             <input type="text" name="full_name" id="" placeholder="Full Name" required>
-            <input type="email" name="" id="" placeholder="Email Address" required>
-            <textarea name="" id="" rows="5" placeholder="Enter Text Here"></textarea>
+            <input type="email" name="email" id="" placeholder="Email Address" required>
+            <textarea name="message" id="" rows="5" placeholder="Enter Text Here"></textarea>
```

Comment:
`textarea` should have both `maxlength` and `required` attributes with `maxlength`  equal to 500.